Sources/MessagingInApp/State/InAppMessageMiddleware.swift

```diff
@@ -65,7 +65,7 @@ func modalMessageDisplayStateMiddleware(logger: Logger, threadUtil: ThreadUtil)
             return next(.reportError(message: "Blocked loading message: \(message.describeForLogs) because another message is currently displayed or cancelled: \(currentMessage)"))
         }
 
-        logger.logWithModuleTag("Showing message: \(message.describeForLogs) with position: \(String(describing: position))", level: .debug)
+        logger.logWithModuleTag("Showing message: \(message) with position: \(String(describing: position))", level: .debug)
```

Comment:
message would already show position now?

Reply:
Removed position from action as it is no longer needed